src/renderer/store/modules/utils.js

```diff
@@ -260,6 +260,28 @@ const actions = {
     return extractors.reduce((a, c) => a || c(), null) || false
   },
 
+  getChannelIdFromUrl (_, url) {
+    /** @type {URL} */
+    let urlObject
+    try {
+      urlObject = new URL(url)
+    } catch (e) {
+      return false
+    }
+
+    const extractors = [
+      // /channel/{channelID}
+      function() {
+        if (urlObject.pathname.startsWith('/channel/') && !urlObject.pathname.endsWith('/channel/')) {
+          // Remove `/channel/` from `/channel/{channelID}`
+          return urlObject.pathname.slice('/channel/'.length)
+        }
+      }
+    ]
+
+    return extractors.reduce((a, c) => a || c(), null) || false
+  },
+
```

Comment:
Good idea, unfortunately 2 out of 4 channel links are not covered by this.
Channel links can be constructed the following ways:
/channel/id
/user/userName
/c/name
/name

That is the reason @Svallinn wanted to reuse the existing extraction code in #1218